### ISSUE TITLE
Need different behavior for /files/upload endpoint

### ISSUE THREAD

**Issue by rayjlinden** [moov-io/paygate#239]
The current behavior of /files/upload basically cuts short the ACH_FILE_TRANSFER_INTERVAL and does the next merge cycle.  This is very useful for testing.

However, this means that some transactions still will not be written out to the sftp server because a full file is not ready based on the the number of lines in the file determined by ACH_FILE_MAX_LINES.

It would be very useful to have an endpoint that flushed all transactions out to ach files more mimicking what would happen at the cut off period for the given routing number.

The main reason we would like such an end point is for testing.  ACH is obviously batch oriented  and ships things once a day.  However, we can't have testing happen only once a day.  We need to be able to mimic the batch cycles in testing code to then validate the transactions we send into the system result in the correct ACH files being generated.

**Comment by adamdecaf** [comment]
@rayjlinden these new endpoints are merged into master. Let me know if you have questions or run into issues. Thanks!
